tests/cases/fourslash/quickInfoForContextuallyTypedParameters.ts

```diff
@@ -34,7 +34,7 @@ verify.quickInfos({
     func?: Function;
 }, obj: T, key: "name"): "name"`,
     4: `function foof<Error, "name">(settings: (row: Error) => {
-    value: Error["name"];
+    value: string;
```

Comment:
Weird - why's this one change? Isn't `Error` in scope?

Reply:
It's not about `Error`. Before when we saw a type parameter that needed mapping, I would fall back to type serialiation. Then type serialization would map `T` to `Error` and then print `Error`. Now when I see the `T` in `T["name"]` and that it can't be reused without falling back to serialization, I fall back to serializaing `T["name"]` insead which gets us `string`.